generator/internal/stub_generator.cc

```diff
@@ -211,33 +216,62 @@ Status StubGenerator::GenerateHeader() {
     "\n"
     "class Default$stub_class_name$ : public $stub_class_name$ {\n"
     " public:");
+  std::unordered_map<std::string, std::string> mixin_grpc_stubs;
+  for (auto const& mixin_method : MixinMethods()) {
+    mixin_grpc_stubs[mixin_method.grpc_stub_name] = mixin_method.grpc_stub_fqn;
```

Comment:
This is strange to me. This information is contained in `MixinMethods()`, so I am not sure why we need another data structure

Reply:
I need this map because I need to deduplicate the grpc stubs, `GetLocation` and `ListLocations` will need the same grpc stub.

Reply:
But let me try to set it to be service and method variables.

Reply:
> I need to deduplicate the grpc stubs

Ah, I see it now. Thanks. You are right. This map is necessary.

My advice is now to loop over `mixin_grpc_stubs` whenever you need to print something.

Reply:
Done.